wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/wsplugins/PluginFQNParserTest.java

```diff
@@ -28,7 +28,10 @@
 import java.util.Map;
 import org.eclipse.che.api.workspace.server.devfile.FileContentProvider;
 import org.eclipse.che.api.workspace.server.spi.InfrastructureException;
+import org.eclipse.che.api.workspace.server.wsplugins.model.Exec;
```

Comment:
looks like still some issues here - the only change in the file is extra imports :man_shrugging:

Reply:
fixed